src/please.go

```diff
@@ -636,7 +636,7 @@ var buildFunctions = map[string]func() int{
 	"format": func() int {
 		if changed, err := format.Format(config, opts.Format.Args.Files.AsStrings(), opts.Format.Write, opts.Format.Quiet); err != nil {
 			log.Fatalf("Failed to reformat files: %s", err)
-		} else if changed && !opts.Format.Write {
+		} else if changed && !opts.Format.Write && opts.Format.Quiet {
```

Comment:
Hmm I think --quiet implies that we're not writing, and we just want to produce an exit code. Perhaps the correct thing to do would be to consider `plz fmt --write --quiet` invalid and just print usage?

Reply:
that makes sense to me, will change